onnxruntime/core/providers/cpu/quantization/quantize_linear.cc

```diff
@@ -19,12 +20,22 @@ class DequantizeLinear final : public OpKernel {
     if (!info.GetAttr<int64_t>("axis", &axis_).IsOK()) {
       axis_ = 1;
     }
+
+    if (!info.GetAttr<int64_t>("block_size", &block_size_).IsOK()) {
```

Comment:
isn't block size is supported from the same opset as int4?

Reply:
[https://github.com/onnx/onnx/blob/main/docs/Changelog.md#DequantizeLinear-21](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#DequantizeLinear-21)

[https://github.com/onnx/onnx/blob/main/docs/Changelog.md#DequantizeLinear-19](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#DequantizeLinear-19)

Reply:
Discussed offline and decided to implement block quantization in a follow-up PR